src/pages/login.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  MDBContainer,
+  MDBRow,
+  MDBCol,
+  MDBCard,
+  MDBCardBody,
+  MDBInput,
+  MDBBtn,
+  MDBIcon,
+  MDBModalFooter,
+} from 'mdbreact';
+import './login.css';
+
+const FormPage = () => {
```

Comment:
Change this to a normal function for consistence reasons only 
```suggestion
function FormPage () {
```